modules/standard/Regexp.chpl

```diff
@@ -18,12 +18,302 @@
  */
 
 /*
-   Regexp Module
+
+This module provides regular expression support.
+
+The regular expression support is built on top of the RE2 regular expression
+library. As such, the exact regular expression syntax available is the syntax
+from RE2, which is available at http://code.google.com/p/re2/wiki/Syntax and
```

Comment:
You'll probably want to insert the rst link stuff for this link, as mentioned here: https://github.com/chapel-lang/chapel/blob/master/doc/release/technotes/README.chpldoc.rst#external-links